spec/helpers/application_helper/buttons/vm_vnc_console_spec.rb

```diff
@@ -30,13 +34,14 @@
 
       context 'and vendor api is not supported' do
         let(:api_version) { 6.5 }
-        it_behaves_like 'a disabled button', 'VNC consoles are unsupported on VMware ESXi 6.5 and later.'
+        include_examples 'ApplicationHelper::Button::Basic disabled',
+                         'VNC consoles are unsupported on VMware ESXi 6.5 and later.'
       end
       context 'and vendor api is supported' do
         let(:api_version) { 6.4 }
 
-        it_behaves_like 'vm_console_with_power_state_on_off', "The web-based VNC console is not available because \
-the VM is not powered on"
+        it_behaves_like 'vm_console_with_power_state_on_off',
```

Comment:
This will be removed in a future commit when refactoring the VM console shared examples.